README.md

```diff
@@ -200,13 +200,11 @@ defer {
     try? group.syncShutdownGracefully()
 }
 
-api.execute(
+try await api.execute(
```

Comment:
Could you replicate the functionality ie
```swift
let result = try await api.execute(...
print(result)
```
Or does that not make sense

Reply:
Whoops, didn't mean to remove the print. Thanks for catching this!